packages/ckeditor5-html-support/tests/manual/htmlcomment.md

```diff
@@ -1 +1,13 @@
 ## HTML comment
+
+1. Toggle the source editing mode and verify that all comments are present. There should be 10 comments.
+1. Open CKEditor 5 inspector and verify, that there are 10 $root attributes, one for each comment. The value set for each attribute is the comment's content (from `C1` to `C10`).
```

Comment:
We should focus here on the functionalities, not the implementation. This test is rather unnecessary.

Reply:
Done.